### DISCUSSION TITLE
How to display/receive blocked mail?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#483] [Q&A] [answered]
I'm not sure how some mail was blocked, because no mail should have been sent to the few deleted addresses that I have. I'm quite confident of this.

![image](https://github.com/anonaddy/anonaddy/assets/42837531/44cef2d3-578f-4b59-ac3f-6e29e1acbf6a)

To ensure that I haven't somehow missed something important, how can I view the blocked mail? I don't mind if I have to just forward all of it to an address of my choice.

---

If this isn't possible, note that since https://app.anonaddy.com/failed-deliveries already exists, the GUI necessary for this already exists.

**Comment by willbrowningme** [comment, marked as answer]
The `blocked` count for an alias is incremented if that alias is either deactivated or deleted.

You must have either previously had or currently have a deactivated and/or deleted alias.

Blocked emails are never accepted into the mail server so it is not possible to view those messages.

**Reply by roke-julian-lockhart** [reply]
@willbrowningme,

> The blocked count for an alias is incremented if that alias is either deactivated or deleted.

So even if I re-enable them, the count won't decrease? I'm not entirely sure what it measures then, if not the amount of individual blocked messages sent via e-mail.

> You must have either previously had or currently have a deactivated and/or deleted alias.

Indeed, I had. (I've re-enabled them now though.)

> Blocked emails are never accepted into the mail server so it is not possible to view those messages.

If I opened an issue about that, would it be considered? I'd like to be able to at least check that spam e-mail is indeed spam, even if it's only held by the server for a maximum of a day, perhaps.

**Comment by willbrowningme** [comment]
No the count will not decrease. It is an ongoing total count of all emails that have been blocked by each alias.

If you want to see all emails (and check if they are indeed spam) then don't deactivate or delete any of your aliases.

**Reply by roke-julian-lockhart** [reply]
Considering, then, @willbrowningme, that it counts the amount of aliases deactivated (if I've interpreted everything correctly) could it be renamed to "Deactivated addresses"?

Yeah, I won't disable them again. That was dumb.
